Microsoft.Toolkit.Uwp.SampleApp/landingPageLinks.json

```diff
@@ -63,6 +63,28 @@
           "url": "https://github.com/telerik/ui-for-uwp"
         }
       ]
+    },
+    {
+      "title": "About the App",
+      "id": "app",
+      "links": [
+        {
+          "title": "Version",
+          "url": "http://aka.ms/uwptoolkit"
+        },
+        {
+          "title": "Newtonsoft Json (Dependency)",
+          "url": "http://www.newtonsoft.com/json"
+        },
+        {
+          "title": "ColorCode-Universal (Dependency)",
+          "url": "https://github.com/WilliamABradley/ColorCode-Universal"
+        },
+        {
```

Comment:
For completeness Monaco link too? https://github.com/hawkerm/monaco-editor-uwp

Reply:
Done